Exercise_4.py

```diff
@@ -1,10 +1,43 @@
-# Python program for implementation of MergeSort 
+"""
```

Comment:
Kindly try to submit all the problems together.

Reply:
Sure, I will do that from further assignments. Sorry for the confusion.